### ISSUE TITLE
Failed Login Limit

### ISSUE THREAD

**Issue by modedeep** [devgeeks/Encryptr#212]
Failed login limit and lock account for a duration of time (20 mins, 30 mins and so on)

If Encryptr doesn't have this yet, i think it's a good idea to add this feature for security reasons. And maybe add camera capture on set failed login limit.

It's better to be safe than sorry.

**Comment by helveticade** [comment]
Agreed that Encryptr needs this. We’ll look into it for a 3.0 release.

**Comment by modedeep** [comment]
Thanks @helveticade. Yes it is a must  feature. Great,  this is a nice progress. Thanks!

**Comment by MichaelJAllen** [comment]
IMHO, this is the most serious issue with Encryptr right now. Given that users are encouraged to enter site URLs, usernames, passwords, credit card details, etc. and put them in a publicly accessible location, it seems crazy that anyone can make multiple attempts to gain access to such an account without any restriction or limitation, and without the owner of the account being notified.

Some form of 2FA/MFA (as outlined in issue  #123) would go a long way to address this, but wouldn't remove the requirement to at least delay a potential intruder.
